Source/inv.cpp

```diff
@@ -865,6 +866,82 @@ BOOL GoldAutoPlace(int pnum)
 
 	return done;
 }
+#else
+BOOL GoldAutoPlace(int pnum)
+{
+	int ii;
+	int xx, yy;
+	int max_gold, gold;
+	BOOL done;
+
+	done = FALSE;
+	for (int i = 0; i < plr[pnum]._pNumInv && !done; i++) {
+		if (plr[pnum].InvList[i]._itype == ITYPE_GOLD) {
+			gold = plr[pnum].InvList[i]._ivalue + plr[pnum].HoldItem._ivalue;
+			if (gold <= MaxGold) {
+				plr[pnum].InvList[i]._ivalue = gold;
+				if (gold >= GOLD_MEDIUM_LIMIT)
+					plr[pnum].InvList[i]._iCurs = ICURS_GOLD_LARGE;
+				else {
+					if (gold <= GOLD_SMALL_LIMIT)
+						plr[pnum].InvList[i]._iCurs = ICURS_GOLD_SMALL;
+					else
+						plr[pnum].InvList[i]._iCurs = ICURS_GOLD_MEDIUM;
+				}

```

Comment:
Would this still be bin exact with the following?

```diff
 				if (gold >= GOLD_MEDIUM_LIMIT)
 					plr[pnum].InvList[i]._iCurs = ICURS_GOLD_LARGE;
-				else {
-					if (gold <= GOLD_SMALL_LIMIT)
-						plr[pnum].InvList[i]._iCurs = ICURS_GOLD_SMALL;
-					else
-						plr[pnum].InvList[i]._iCurs = ICURS_GOLD_MEDIUM;
-				}
+				else if (gold <= GOLD_SMALL_LIMIT)
+					plr[pnum].InvList[i]._iCurs = ICURS_GOLD_SMALL;
+				else
+					plr[pnum].InvList[i]._iCurs = ICURS_GOLD_MEDIUM;
+				}
```

Reply:
Yes. Applied the changes.